README.md

```diff
@@ -1,2 +1,67 @@
-# election-manager
-Manages election configuration, ballot creation, and more.
+# Election Manager
+
+This app is intended to be used on an offline computer by an election admin.
+
+### Features
+- manage election configuration
+- create hand-marked paper ballots
+- export an "election ballot package" for [Mail Ballot Manager](https://github.com/votingworks/mail-ballot-manager)
+
+## Workflow
+
+1. Admin may create a new election or select an existing election configuration file (`election.json`) to start with (or possibly a previously created election ballot package). 
+2. Admin can edit the election configuration via simple UI forms at any time.
+3. Admin can list all ballots by precinct/style.
+4. Admin can view an individual ballot, both HTML and PDF versions.
+5. Admin can export an election ballot package for [Mail Ballot Manager](https://github.com/votingworks/mail-ballot-manager).
+
+   Election Ballot Package contains:
+     - `election.json` - VotingWorks data format for election data: contests, canidates,ballot styles, precincts, etc.
+     - approved official ballots in pdf format:
+       - one for each combination of ballot style and precinct
+       - file name pattern: `ballot-style-7-precinct-8.pdf`
```

Comment:
can we build an election ID into the filename too?

Reply:
Would this election id be a top-level key in the `election.json` file or something generated from other data in the election configuration?

Reply:
it could be a hash of `election.json`, which would be long, but would certainly prevent any confusion.